From: Zachary Cook <zachcook1991@gmail.com>
Date: Thu, 6 Mar 2014 03:49:49 -0500
Subject: man: systemd-bootchart - fix spacing in command

Use the same formatting as the systemd-analyze man page, so that man shows a space.

(cherry picked from commit 82ed60080d327d7301fcd55f5a1f8511f894b9d5)
(cherry picked from commit 7117b40be2484d2f318c14d42b8c1f47d44a4465)
---
 man/systemd-bootchart.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/man/systemd-bootchart.xml b/man/systemd-bootchart.xml
index ae432b5..818afa1 100644
--- a/man/systemd-bootchart.xml
+++ b/man/systemd-bootchart.xml
@@ -76,8 +76,8 @@
                         in which order, and where possible problems
                         exist in the startup sequence of the system.
                         It is essentially a more detailed version of
-                        the <command>systemd-analyze</command>
-                        <command>plot</command> function.
+                        the <command>systemd-analyze plot</command>
+                        function.
                 </para>
                 <para>
                         Of course, bootchart can also be used at any
